Parse-Dashboard/Authentication.js

```diff
@@ -96,12 +97,14 @@ function authenticate(userToTest, usernameOnly) {
         matchingUsername = user.user;
         // User restricted apps
         appsUserHasAccessTo = user.apps || null;
+        isReadOnly = user.readOnly;
       }
 
       return isAuthenticated;
     }) ? true : false;
```

Comment:
ewww, remove the `? true: false`? as completely redundant?

Reply:
yeap